src/commandHandlers/issue.ts

```diff
@@ -0,0 +1,60 @@
+import axios, { AxiosRequestConfig } from 'axios';
+import { Message, MessageEmbed } from 'discord.js';
+
+const req = async(q: string): Promise<any[]> => {
+    const config: AxiosRequestConfig = {
+        url: `https://api.github.com/search/repositories`,
```

Comment:
This might be better in the config.ts

Reply:
I'm really new to PRs so I don't know how you guys usually do it. Since the liner didn't yell at me I thought it was OK, but I'll do those changes tomorrow, thanks!

Reply:
Apparently I commented in the wrong place thanks to mobile github...
How would it improve by being on the config file?

Reply:
You can add this on the [config.ts](https://github.com/EddieJaoudeCommunity/EddieBot/blob/develop/src/config.ts) file and then import it on that `issue.ts` file:
```ts
export const githubSearchRepositoriesUrl = 'https://api.github.com/search/repositories';
```

You can name it something else if you prefer. I tend to prefer explicit names even if they are longer, without abbreviations 😃.

Reply:
I see that's better than what I did. I'll change it on the next push 😃.